lib/test_suite.py

```diff
@@ -289,3 +289,7 @@ def fragile_retries(self):
 
     def show_reproduce_content(self):
         return self.ini['show_reproduce_content']
+
+    def check_is_long(self, test):
```

Comment:
'Check' is the too common word in most contexts and means nothing. I would name the function like `test_is_long()`.

Reply:
Right, that's better, changed.